sdk/textanalytics/Azure.AI.TextAnalytics/tests/AnalyzeSentimentTests.cs

```diff
@@ -270,6 +270,23 @@ public async Task AnalyzeSentimentBatchConvenienceWithStatisticsTest()
             Assert.IsNotNull(results.Statistics.InvalidDocumentCount);
         }
 
+        [Test]
+        public async Task AnalyzeSentimentBatchConvenienceWithStringIndexTypeTest()
```

Comment:
I am not sure how to test as `stringindextype` is not returned in the response.

Reply:
You could do 2 things:
- Add more tests [here](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/textanalytics/Azure.AI.TextAnalytics/tests/TextAnalyticsClientLiveTests.cs) with a different encoding and make sure the service is returning the offset corresponding to the encoding
- Add tests in our [mock clas](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/textanalytics/Azure.AI.TextAnalytics/tests/TextAnalyticsClientMockTests.cs)s where you can hardcode the request and then you can [query the headers](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/formrecognizer/Azure.AI.FormRecognizer/tests/FormRecognizerClient/FormRecognizerClientMockTests.cs#L65) and see if the stringIndexType was set to what you specified.

Reply:
for option 1, maybe better to make it its own class. See what Python did: https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/textanalytics/azure-ai-textanalytics/tests/test_encoding.py we did the same but in the TextAnalyticsTest class so maybe move/rename the file?

Reply:
that way these tests you are adding are not neccesary